code/game/turfs/simulated/wall_types.dm

```diff
@@ -110,12 +110,6 @@
 	. = ..(mapload, MATERIAL_TITANIUM, MATERIAL_TITANIUM)
 	canSmoothWith = list(src.type)
 
-/turf/simulated/wall/gold_reinforced/Initialize(mapload)
-	canSmoothWith = list(src.type)
-	. = ..(mapload, MATERIAL_GOLD, MATERIAL_PLASTEEL)
-	canSmoothWith = list(src.type)
-	color = COLOR_GOLD
```

Comment:
why is this being removed?

Reply:
I added it as part of this PR, but the places it was used were since replaced